handler/src/main/java/io/netty/handler/ssl/SslProvider.java

```diff
@@ -27,5 +29,9 @@
     /**
      * OpenSSL-based implementation.
      */
-    OPENSSL
+    OPENSSL,
+    /**
+     * OpenSSL-based implementation which does not have finalizers and instead implements {@link ReferenceCounted}.
+     */
+    OPENSSL_RELEASABLE
```

Comment:
Yip.

Actually annotations are not inherited by default (unless the `@Inherited` meta annotation is present) [1]. I verified this using reflection and changing the retention policy on that `UnstableApi` so I will add the `UnstableApi` to these classes.

[1] http://stackoverflow.com/a/10082663/3993966